.github/pull_request_template.md

```diff
@@ -1,6 +1,5 @@
 Please make sure to document all user-facing changes in the `CANGELOG.md` file.
```

Comment:
Come to think of it, we could also tell people to leave a PR description to help potential reviewers. PRs without descriptions make me sad. Maybe something like:

> Please make sure to document all user-facing changes in the `CANGELOG.md` file, and leave a useful description in the Pull Request itself.

Reply:
Well the commit message and changes should usually suffice. I don't remember any PRs where I was questioning the reason why it was created and thought that a description would help.

Reply:
yea, fair enough. As long as the commit messages aren't, `do the thing` :P